test_elasticsearch/test_client/test_utils.py

```diff
@@ -17,3 +17,24 @@ def test_handles_utf_encoded_string(self):
         id = "中文".encode('utf-8')
         self.assertEquals('/some-index/type/%E4%B8%AD%E6%96%87', _make_path('some-index', 'type', id))
 
+
+class TestEscape(TestCase):
+    def test_handles_ascii(self):
+        string = "abc123"
+        self.assertEquals(
+            b'abc123',
+            _escape(string)
+        )
+    def test_handles_unicode(self):
+        string = "中文"
+        self.assertEquals(
+            b'\xe4\xb8\xad\xe6\x96\x87',
+            _escape(string)
+        )
+
+    def test_handles_bytestring(self):
```

Comment:
yeah I was running this in 2 different envs (python2 and python3) to make sure it worked in both. I just wanted to have an explicit test case for handling bytestring.